### DISCUSSION TITLE
"error": "SOCKS5 request failed: 4"是什么意思?

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [apernet/hysteria#932] [Q&A] [answered]
"error": "SOCKS5 request failed: 4"是什么意思?

**Comment by haruue** [comment, marked as answer]
你的服务端配置了 socks5 outbounds 吧。
这个是你配置的上游 socks5 服务器返回的错误， 根据 [RFC 1928](https://datatracker.ietf.org/doc/html/rfc1928#section-6)， 4 是 `X'04' Host unreachable`
